proto/slinky/marketmap/v1/market.proto

```diff
@@ -9,6 +9,9 @@ import "gogoproto/gogo.proto";
 // feed is scaled to a number of decimal places and has a minimum number of
 // providers required to consider the ticker valid.
 message Ticker {
+  option (gogoproto.goproto_stringer) = false;
```

Comment:
What about min_ppm?

Idk if we want to add now, but maybe we should make a TODO or note it

Reply:
we dont have any of that built out rn, so i prefer not adding for the time being. can make a linear task for it.